bool turtlebot_enable
bool fwd_enable
bool follow_enable
bool right_enable
bool left_enable
